### ISSUE TITLE
Cannot update MAL due to invalid field

### ISSUE THREAD

**Issue by xenoryt** [m4tx/tundra#113]
I see the following errors
```
2024-01-28 14:02:23 [INFO] [tundra::app] Found an active player: SMPlayer, playing Hataraku Maou-sama season 1 episode 4
2024-01-28 14:02:23 [ERROR] [tundra::gtk_gui] Could not communicate with MAL: error decoding response body: unknown variant `cm`, expected one of `tv`, `ova`, `movie`, `special`, `ona`, `music`, `unknown` at line 1 column 4499
2024-01-28 14:02:23 [ERROR] [tundra::gtk_gui] error decoding response body: unknown variant `cm`, expected one of `tv`, `ova`, `movie`, `special`, `ona`, `music`, `unknown` at line 1 column 4499
```

Name of file: `Hataraku_Maou-sama_-_04.mp4`

Tundra version: 0.5.1

Since the error mentions "response body" I thought it might have updated MAL. But I have double checked MAL and the watched episode is still set to 3.

**Comment by m4tx** [comment]
@xenoryt please let me know if this is still happening with Tundra 0.5.2, hopefully this should have been fixed in 11efdd1cf9c9198c893219f498e40bb348d099f6.

**Comment by Noranecore** [comment]
I'm getting a similar error.
```
2024-01-31 14:49:00 [INFO] [tundra::app] Found an active player: mpv, playing Youkoso Jitsuryoku Shijou Shugi no Kyoushitsu e Season 3 season 1 episode 5
2024-01-31 14:49:00 [ERROR] [tundra::app] Could not communicate with MAL: error decoding response body: unknown variant `pv`, expected one of `tv`, `ova`, `movie`, `special`, `ona`, `music`, `cm`, `unknown` at line 1 column 2623

Caused by:
    0: error decoding response body: unknown variant `pv`, expected one of `tv`, `ova`, `movie`, `special`, `ona`, `music`, `cm`, `unknown` at line 1 column 2623
    1: unknown variant `pv`, expected one of `tv`, `ova`, `movie`, `special`, `ona`, `music`, `cm`, `unknown` at line 1 column 2623
```
I'm using the ani-cli script.

**Comment by m4tx** [comment]
Interesting that these media types are not documented in the MyAnimeList API... Should be fixed in b9d46b5bb4c6fae98f2a8606dbc29ef80d3fa3b6. @Noranecore could you please try with version 0.5.3?

**Comment by Noranecore** [comment]
> Interesting that these media types are not documented in the MyAnimeList API... Should be fixed in [b9d46b5](https://github.com/m4tx/tundra/commit/b9d46b5bb4c6fae98f2a8606dbc29ef80d3fa3b6). @Noranecore could you please try with version 0.5.3?

working for me now

**Comment by m4tx** [comment]
Thanks for confirming, I'll close the issue now. Feel free to comment/raise a new issue if the problem appears again.
